googledrive/src/main/java/org/assimbly/googledrive/GoogleDriveEndpoint.java

```diff
@@ -22,6 +24,8 @@
 @SuppressWarnings("PackageAccessibility")
 public class GoogleDriveEndpoint extends ProcessorEndpoint {
 
+    private static final Logger LOG = LoggerFactory.getLogger(GoogleDriveEndpoint.class);
+
```

Comment:
There's a more generic (and I think recommended way) to get the logger:

	protected Logger log = LoggerFactory.getLogger(getClass());

Reply:
deleted line. don't need anymore